diff --git a/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts b/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts
index ffedb9a..605dac2 100644
--- a/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts
+++ b/node_modules/@snapshot-labs/lock/connectors/walletconnect.ts
@@ -7,7 +7,7 @@ export default class Connector extends LockConnector {
     let provider;
     try {
       const WalletConnectProvider = (await get()).default;
-      provider = new WalletConnectProvider(this.options);
+      provider = new WalletConnectProvider.default(this.options);
       await provider.enable();
     } catch (e) {
       console.error(e);
