diff --git a/node_modules/available-typed-arrays/index.js b/node_modules/available-typed-arrays/index.js
index 00cb9c1..16e8de9 100644
--- a/node_modules/available-typed-arrays/index.js
+++ b/node_modules/available-typed-arrays/index.js
@@ -15,11 +15,5 @@ var possibleNames = [
 ];
 
 module.exports = function availableTypedArrays() {
-	var out = [];
-	for (var i = 0; i < possibleNames.length; i++) {
-		if (typeof global[possibleNames[i]] === 'function') {
-			out[out.length] = possibleNames[i];
-		}
-	}
-	return out;
+	return [];
 };
