diff --git a/node_modules/cipher-base/index.js b/node_modules/cipher-base/index.js
index 6728005..976a8e3 100644
--- a/node_modules/cipher-base/index.js
+++ b/node_modules/cipher-base/index.js
@@ -1,5 +1,5 @@
 var Buffer = require('safe-buffer').Buffer
-var Transform = require('stream').Transform
+const { Transform } = require('readable-stream')
 var StringDecoder = require('string_decoder').StringDecoder
 var inherits = require('inherits')
 
