diff --git a/node_modules/ethereumjs-tx/es5/index.js b/node_modules/ethereumjs-tx/es5/index.js
index 0108471..270427e 100644
--- a/node_modules/ethereumjs-tx/es5/index.js
+++ b/node_modules/ethereumjs-tx/es5/index.js
@@ -4,7 +4,7 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
 
 var ethUtil = require('ethereumjs-util');
 var fees = require('ethereum-common/params.json');
-var BN = ethUtil.BN;
+var { BN } = ethUtil;
 
 // secp256k1n/2
 var N_DIV_2 = new BN('7fffffffffffffffffffffffffffffff5d576e7357a4501ddfe92f46681b20a0', 16);
