diff --git a/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/fake.js b/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/fake.js
index 8f4c4cb..d3e4b22 100644
--- a/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/fake.js
+++ b/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/fake.js
@@ -13,7 +13,7 @@ var __extends = (this && this.__extends) || (function () {
     };
 })();
 Object.defineProperty(exports, "__esModule", { value: true });
-var ethereumjs_util_1 = require("ethereumjs-util");
+var ethereumjs_util_1 = window.require("ethereumjs-util");
 var buffer_1 = require("buffer");
 var transaction_1 = require("./transaction");
 /**
diff --git a/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/transaction.js b/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/transaction.js
index 541962c..cd8d872 100644
--- a/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/transaction.js
+++ b/node_modules/ethereumjs-vm/node_modules/ethereumjs-tx/dist/transaction.js
@@ -11,7 +11,7 @@ var __assign = (this && this.__assign) || function () {
     return __assign.apply(this, arguments);
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-var ethereumjs_util_1 = require("ethereumjs-util");
+var ethereumjs_util_1 = window.require("ethereumjs-util");
 var ethereumjs_common_1 = require("ethereumjs-common");
 var buffer_1 = require("buffer");
 // secp256k1n/2
