diff --git a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
index 192d451..bf3e60a 100644
--- a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
+++ b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_readable.js
@@ -45,7 +45,7 @@ var Stream = require('./internal/streams/stream');
 
 var Buffer = require('buffer').Buffer;
 
-var OurUint8Array = global.Uint8Array || function () {};
+var OurUint8Array = function () {};
 
 function _uint8ArrayToBuffer(chunk) {
   return Buffer.from(chunk);
diff --git a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_writable.js b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_writable.js
index a2634d7..5bffa2e 100644
--- a/node_modules/hash-base/node_modules/readable-stream/lib/_stream_writable.js
+++ b/node_modules/hash-base/node_modules/readable-stream/lib/_stream_writable.js
@@ -69,7 +69,7 @@ var Stream = require('./internal/streams/stream');
 
 var Buffer = require('buffer').Buffer;
 
-var OurUint8Array = global.Uint8Array || function () {};
+var OurUint8Array = function () {};
 
 function _uint8ArrayToBuffer(chunk) {
   return Buffer.from(chunk);
