diff --git a/node_modules/idna-uts46-hx/uts46.js b/node_modules/idna-uts46-hx/uts46.js
index ece002f..fab36ab 100644
--- a/node_modules/idna-uts46-hx/uts46.js
+++ b/node_modules/idna-uts46-hx/uts46.js
@@ -12,7 +12,7 @@
     root.uts46 = factory(root.punycode, root.idna_map);
   }
 }(this, function(punycode, idna_map) {
-
+  punycode = punycode && punycode.hasOwnProperty('default') ? punycode.default : punycode;
   function mapLabel(label, useStd3ASCII, transitional) {
     var mapped = [];
     var chars = punycode.ucs2.decode(label);
