diff --git a/node_modules/util/util.js b/node_modules/util/util.js
index 6eea657..1fcaa9c 100644
--- a/node_modules/util/util.js
+++ b/node_modules/util/util.js
@@ -106,14 +106,6 @@ exports.deprecate = function(fn, msg) {
 var debugs = {};
 var debugEnvRegex = /^$/;
 
-if (process.env.NODE_DEBUG) {
-  var debugEnv = process.env.NODE_DEBUG;
-  debugEnv = debugEnv.replace(/[|\\{}()[\]^$+?.]/g, '\\$&')
-    .replace(/\*/g, '.*')
-    .replace(/,/g, '$|^')
-    .toUpperCase();
-  debugEnvRegex = new RegExp('^' + debugEnv + '$', 'i');
-}
 exports.debuglog = function(set) {
   set = set.toUpperCase();
   if (!debugs[set]) {
